core/src/test/java/com/klaytn/caver/scenario/RoleBasedAccountValueTransferIT.java

```diff
@@ -0,0 +1,186 @@
+package com.klaytn.caver.scenario;
+
+import com.klaytn.caver.methods.response.KlayTransactionReceipt;
+import com.klaytn.caver.tx.model.TransactionTransformer;
+import com.klaytn.caver.tx.model.ValueTransferTransaction;
+import com.klaytn.caver.tx.type.TxTypeFeeDelegate;
+import org.junit.Test;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+import java.util.Random;
+
+import static com.klaytn.caver.base.Accounts.BRANDON;
+import static org.junit.Assert.assertEquals;
+
+public class RoleBasedAccountValueTransferIT extends RoleBasedAccountScenario {
+
+    private final ReceiptChecker getReceiptChecker = (TransactionTransformer transactionTransformer, KlayTransactionReceipt.TransactionReceipt transactionReceipt) -> {
+        assertEquals("0x1", transactionReceipt.getStatus());
+    };
+
+    private final ReceiptChecker getFeeDelegatedValueTransferMemoWithRatioReceiptChecker = (TransactionTransformer transactionTransformer, KlayTransactionReceipt.TransactionReceipt transactionReceipt) -> {
+        assertEquals("0x1", transactionReceipt.getStatus());
+        assertEquals(Numeric.toHexString(MEMO.getBytes()), transactionReceipt.getInput());
+        TxTypeFeeDelegate txTypeFeeDelegate = (TxTypeFeeDelegate) transactionTransformer.build();
+        assertEquals(Numeric.toHexStringWithPrefix(txTypeFeeDelegate.getFeeRatio()), transactionReceipt.getFeeRatio());
+    };
+
+    private final ReceiptChecker getFeeDelegatedValueTransferWithRatioReceiptChecker = (TransactionTransformer transactionTransformer, KlayTransactionReceipt.TransactionReceipt transactionReceipt) -> {
+        assertEquals("0x1", transactionReceipt.getStatus());
+        TxTypeFeeDelegate txTypeFeeDelegate = (TxTypeFeeDelegate) transactionTransformer.build();
+        assertEquals(Numeric.toHexStringWithPrefix(txTypeFeeDelegate.getFeeRatio()), transactionReceipt.getFeeRatio());
+    };
+
+    private final ReceiptChecker getFeeDelegatedValueTransferMemoReceiptChecker = (TransactionTransformer transactionTransformer, KlayTransactionReceipt.TransactionReceipt transactionReceipt) -> {
+        assertEquals("0x1", transactionReceipt.getStatus());
+        assertEquals(Numeric.toHexString(MEMO.getBytes()), transactionReceipt.getInput());
+    };
+
+    private final ValueTransferTransaction getValueTransferTransaction(String from) {
+        return ValueTransferTransaction.create(from, BRANDON.getAddress(), BigInteger.ONE, GAS_LIMIT);
+    }
+
+    //////////////////////////////// BasicTest - value Transfer ////////////////////////////////
+    @Test
+    public void valueTransferTest() throws Exception {
+        roleBasedTransactionTest(
+                (String from) -> getValueTransferTransaction(from),
+                getReceiptChecker, false);
```

Comment:
Parameter(`isUpdateTest`) is hard to understand what function of this parameter. If you make setter methods(`isUpdateTest()`, `isTransactionTest()`), you could deliver test purpose more clearly.